wao/scalping_strategy_tester.py

```diff
@@ -52,7 +52,8 @@ def get_year_range() -> str:
 def write_to_csv(list_of_row):
     print("write_to_csv:... ")
     column_title = ['coin', 'Brain', 'human_readable_time', 'timeframe', 'win_rate_percentage_per_day',
-                    'number_of_trades_per_day', 'average_percentage_per_trade', 'cumulative_percentage_per_day']
+                    'number_of_trades_per_day', 'average_percentage_per_trade', 'cumulative_percentage_per_day',
+                    'win_rate_percentage_per_year']
```

Comment:
What if it's three years data , this is confusing 
Rename to: per_year_or_two_or_three 

 if you can - include the number of years dynamically  , not a requirement